src/irmin-pack/irmin_pack_layers.ml

```diff
@@ -954,6 +954,7 @@ struct
     (match heads with [] -> Repo.heads t | m -> Lwt.return m) >>= fun heads ->
     if t.X.Repo.closed then Lwt.fail_with "store is closed"
     else if Pack_config.readonly t.X.Repo.config then raise RO_Not_Allowed
+    else if Lwt_mutex.is_locked freeze_lock then Lwt.return ()
```

Comment:
Suspect it's worth logging at least a warning here (and perhaps even having a different return value).

Reply:
I think we should remove this commit for now, until we figure out a strategy for the bootstrapping. At the moment, if we don't block at all during bootstrapping, the first  freeze will keep on accumulating newies and possibly never complete.